README.md

```diff
@@ -225,14 +225,15 @@ To run a jasmine Webrick server on port 3001, from the console do:
 and then go to `http://localhost:3001/` in your browser.  Whenever you reload the page, the tests are run again.
 
 
-Sample data
+Create an initial admin account
 -----------
 
-To create a sample admin account for development and testing:
+To create an initial manager admin account, from the console do:
 
-    rake primary_source_sets:samples:create_admin
+    Admin.new({ email: '[EMAIL]', password: '[PASSWORD]', password_confirmation: '[PASSWORD]', status: 2 }).save
```

Comment:
Can you provide an example so it's clear that the email, password and password confirmation are not intended to be wrapped in brackets?